pkg/R/utils.R

```diff
@@ -419,3 +419,82 @@ cleanClosure <- function(func, checkedFuncs = new.env()) {
   }
   func
 }
+
+# Common function for implmentation of zipRDD() and cartesian().
+# param
+#   x An RDD.
+#   other An RDD.
+#   operation The specified operation, "zip" or "cartesian".
+#
+# return value
+#   The result RDD.
+doZipOrCartesian <- function(x, other, operation = c("zip", "cartesian")) {
```

Comment:
Yeah having one parameter is fine for now. Ideally we'd call it `mergeStrategy` which could be `zip` or `merge` etc. but I'm wondering if that is complicating things too much. I'm fine with calling the variable a boolean `zip` for now.